scripts/cmd.d/list

```diff
@@ -0,0 +1,5 @@
+. ${DEVENVROOT}/scripts/func.d/bash_utils
+
+display -e "$(get_list flavor)"
```

Comment:
It's so simple and may also be just a bash function.  But in the current architecture I am not sure how to add a function as a command.